src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
Consider moving the comments above the relevant code, indented relative to the position of the code, as per the coding standard.